app/data/repositories/CardsRepository.py

```diff
@@ -0,0 +1,72 @@
+from app.data import dbConn
+from app.data.models.Card import Card
+
+
+conn = dbConn.get()
+
+
+def create(deck_id, word, translation):
+
+    deck_id = deck_id
+    word = word
+    translation = translation
+
+    query = f"INSERT INTO card (deck_id, word, translation) VALUES ({deck_id}, '{word}', '{translation}');"
+
+    c = conn.cursor()
+
+    c.execute(query)
+
+    conn.commit()
+
+    if c.lastrowid:
+        return True
+    
+    return False
+
+
+def card(data):
+    return Card.card(data)
+
+
+def getAll():
+
+    query = "SELECT * FROM card"
+
+    c = conn.cursor()
+
+    c.execute(query)
+
+    cards = []
+
+    for data in c.fetchall():
+        cards.append(card(data))
```

Comment:
what is data exactly?

Reply:
c.fetchall is a multi-array of rows from the card table
each row is an array of values from columns
so data looks like [card_id, name, translation]

Reply:
done